framework/src/play/db/jpa/JPAPlugin.java

```diff
@@ -239,39 +239,17 @@ public static String getDefaultDialect(Configuration dbConfig, String driver) {
         } else if ("com.mysql.cj.jdbc.Driver".equals(driver)) {
             return "org.hibernate.dialect.MySQL8Dialect";
         } else if ("com.mysql.jdbc.Driver".equals(driver)) {
-            return "play.db.jpa.MySQLDialect";
+            return "org.hibernate.dialect.MySQLDialect";
         } else if ("org.postgresql.Driver".equals(driver)) {
             return "org.hibernate.dialect.PostgreSQLDialect";
         } else if ("com.ibm.db2.jdbc.app.DB2Driver".equals(driver)) {
             return "org.hibernate.dialect.DB2Dialect";
         } else if ("com.ibm.as400.access.AS400JDBCDriver".equals(driver)) {
             return "org.hibernate.dialect.DB2400Dialect";
-        } else if ("com.ibm.as400.access.AS390JDBCDriver".equals(driver)) {
-            return "org.hibernate.dialect.DB2390Dialect";
         } else if ("oracle.jdbc.OracleDriver".equals(driver)) {
-            return "org.hibernate.dialect.Oracle10gDialect";
-        } else if ("com.sybase.jdbc2.jdbc.SybDriver".equals(driver)) {
-            return "org.hibernate.dialect.SybaseAnywhereDialect";
+            return "org.hibernate.dialect.OracleDialect";
         } else if ("com.microsoft.jdbc.sqlserver.SQLServerDriver".equals(driver)) {
             return "org.hibernate.dialect.SQLServerDialect";
-        } else if ("com.sap.dbtech.jdbc.DriverSapDB".equals(driver)) {
```

Comment:
why did you remove these drivers support?

Reply:
I removed the drivers for which there are no longer any dialects in the Hibernate core.